config/kubernetes/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kbs
+spec:
+  selector:
+    matchLabels:
+      app: kbs
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: kbs
+    spec:
+      containers:
+      - name: kbs
+        image: kbs-container-image
```

Comment:
What's this image used for?

Reply:
So this image is set from the kustomization.yaml file. We can provide a default one as well. But is there a default image that we build with `-as-builtin` ?

Reply:
We can provide config for the grpc one as well. But I thought this was a good start.

Reply:
Sounds good to me. But we do not have a default image with built-in-as as well as a GH Action to build the image. We might need one.

Reply:
@Xynnn007 created an issue for this: https://github.com/confidential-containers/kbs/issues/91

Reply:
@Xynnn007 now that #91 is fixed where will be image be published?

Reply:
https://github.com/confidential-containers/kbs/issues/96

Reply:
FYI: v0.6.0 has released `ghcr.io/confidential-containers/key-broker-service:built-in-as-v0.6.0`

Reply:
Done PTAL.